common/api/core-react.api.md

```diff
@@ -1230,6 +1230,7 @@ export interface PopupProps extends CommonProps {
     onOpen?: () => void;
     onOutsideClick?: (e: MouseEvent) => void;
     onWheel?: (e: WheelEvent) => void;
+    portalTarget?: HTMLElement;
```

Comment:
API addition, document in NextVersion

Reply:
Should be advertise changes to deprecated components?

Reply:
I don't think it's an issue. Changelogs should be complete with all the changes to avoid any surprises.

Reply:
Updated changelog